File: 1020 - Age in Days/Main.js
let n = parseInt(require("fs").readFileSync("/dev/stdin", "utf8"));
console.log(`${Math.floor(n / 365)} ano(s)`);
n %= 365;
console.log(`${Math.floor(n / 30)} mes(es)`);
console.log(`${n % 30} dia(s)`);

File: 1020 - Age in Days/Main.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    cout << n / 365 << " ano(s)\n";
    n %= 365;
    cout << n / 30 << " mes(es)\n";
    cout << n % 30 << " dia(s)\n";
    return 0;
}